src/GraphQL/Types/GraphTypesLookup.cs

```diff
@@ -506,46 +506,42 @@ private void ApplyTypeReference(IGraphType type)
 
             if (type is IObjectGraphType objectType)
             {
-                objectType.ResolvedInterfaces = objectType
```

Comment:
Now there is no need to create a new list and replace the old one with it (with LINQ stuff).